.azure-pipelines/templates/stages/deploy-stage.yml

```diff
@@ -53,10 +54,26 @@ stages:
             inputs:
               secureFile: snapcraft.cfg
           - bash: |
+              set -e
+              retry_command() {
```

Comment:
I put this inline because we're not using it anywhere else. Shouldn't be too bad to extract if we want to use it elsewhere later though.